### DISCUSSION TITLE
Where is $prefix defined??

### DISCUSSION THREAD

**Discussion by Routhinator** [twbs/bootstrap#37218] [General] [answered]
Hey there folks,

I updated to Angular 14 which updated webpack, and now I cannot compile 5.1.3 OR 5.2.1 because `$prefix` is undefined. 

Been digging through the issues for an hour and cannot find anything on this.

**Comment by julien-deramond** [comment, marked as answer]
Hey @Routhinator 
Maybe you have an issue with the way you include the .scss files coming from Bootstrap (see [Customize > Sass > Importing](https://getbootstrap.com/docs/5.2/customize/sass/#importing)).
`$prefix` is defined in __variables.scss_.

**Reply by Routhinator** [reply]
Hmm, interesting. I am not sure how that would be as this breaks by updating angular and the scss module it uses. Doing that does not change the order. This builds in Angular 13 and not in 14. 

I'll play around with it and see.

**Reply by Routhinator** [reply]
I got it sorted now, no idea why this cropped up, but my copy of variables was missing some new vars, solved.
